lang/java/avro/src/main/java/org/apache/avro/SchemaCompatibility.java

```diff
@@ -20,20 +20,19 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.HashMap;
-import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
-
+import java.util.TreeSet;
 import org.apache.avro.Schema.Field;
 import org.apache.avro.Schema.Type;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
- * Evaluate the compatibility between a reader schema and a writer schema.
- * A reader and a writer schema are declared compatible if all datum instances of the writer
```

Comment:
New try, where I manually tried to minimize the formatting differences wrt. master
But, I must say, it would help development a great deal if Avro could settle on a common coding standard.
When reformatting this, I tried following the existing guidelines (Java Lang conventions, and used 100 column width).
Since the early '00's, I have delegated formatting to tools (IDEs). Agreed, not all developers use an IDE, but there are compatibility checkers that can be used to validate compliance (e.g. CheckStyle...)